extern/boostd-data/couchbase/db.go

```diff
@@ -726,7 +726,7 @@ func (db *DB) NextPiecesToCheck(ctx context.Context) ([]cid.Cid, error) {
 }
 
 // The minimum frequency with which to check pieces for errors (eg bad index)
-const MinPieceCheckPeriod = time.Second
+var MinPieceCheckPeriod = 30 * time.Second
```

Comment:
Bumping up to 30 seconds, as we don't want to check pieces every second - even 30 seconds is probably aggressive in the Filecoin realm.

Changing to `var` so that we can update this value in tests.